modules/serverless-overriding-pdbs-eventing.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /eventing/tuning/serverless-ha.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-eventing_{context}"]
+= Overriding disruption budgets
+
+A Pod Disruption Budget (PDB) is a standard feature of Kubernetes APIs that helps limit the disruption to an application when its pods need to be rescheduled for maintenance reasons.
+
+.Procedure
+
+* You can override the default PDB for a specific resource by modifying the `minAvailable` configuration value in the `KnativeEventing CR`.
```

Comment:
Two things here - CR should be placed outside of the backticks, and it should be spelled out since this is the first mention of the term in this section:
```suggestion
* You can override the default PDB for a specific resource by modifying the `minAvailable` configuration value in the `KnativeEventing` custom resource (CR).
```